src/components/PageWarning.tsx

```diff
@@ -0,0 +1,17 @@
+import { FunctionComponent } from 'react';
+
+const PageWarning: FunctionComponent = () => {
```

Comment:
`PageWarning` is a generic name but this component is specific because it has a warning for a document which is no longer valid

I would either
- Rename it to be more specific, for example `DocumentValidityWarning`
- Introduce two props: `title` and `content` and pass them from `[documentSubmissionId].tsx`

Reply:
Good point!